audmetric/core/api.py

```diff
@@ -1186,28 +1186,74 @@ def weighted_confusion_error(
 def word_error_rate(
     truth: Sequence[Sequence[str]],
     prediction: Sequence[Sequence[str]],
+    *,
+    norm: str = "truth",
 ) -> float:
     r"""Word error rate based on edit distance.
 
+    The word error rate is computed
+    by aggregating the normalized edit distances
+    of each (truth, prediction)-pair
+    and averaging the aggregated score
+    by the number of pairs.
+
+    The normalized edit distance
+    of each (truth, prediction)-pair is computed
+    as the edit distance divided by a normalization factor n.
+    This represents the average editing cost per sequence item.
+
+    The value of n depends on the norm parameter:
+
+    If ``norm`` is ``"truth"``,
```

Comment:
I would write it as a normal sentence:

```suggestion
    The value of n depends on the ``norm`` parameter.
    If ``norm`` is ``"truth"``,
```

Reply:
Soory, I changed my mind, and will do another proposal.

Reply:
I suggested https://github.com/audeering/audmetric/pull/63/files#r1879678670 instead.

Reply:
I had it put this way as below there is also the sentence starting with "If ``norm`` is ``"longest"``"
There is a line break after the formula before this sentence. In rst I found it more readable to have a line break.

I am committing the suggestion anyway and see how it comes out in sphinx.

Reply:
Sorry you had been too fast.
I suggested to not apply this as I saw it does not look good and suggested an alternative below.

I will open a new suggestion, sorry for the confusion.